app/models/beatmap.py

```diff
@@ -37,6 +37,7 @@ class Beatmap:
     bpm: int
     filename: str
     frozen: bool
+    rankedby: Optional[str]
```

Comment:
woops `| rankedby              | mediumint    | YES  |     | NULL    |       |`

```suggestion
    rankedby: Optional[int]
```